### PR TITLE
[src] Fix LambdaOverlapAdd

### PR DESCRIPTION
**Pull request by mpariente**
The previous loss function to reorder the chunks was weird, this should fix it. 

Which loss function would make the most sense? Currently it's kind of unnormalized correlation. 

Also, we probably need a better unit test for this. 

@groadabike, could you checkout this branch and check if the reordering works better in your case please? That'd be great

### CODE REVIEW

**Comment by faroit** [comment]
@mpariente while you are at it, it had trouble understanding the purpose of `reorder_chunks` when I glanced at the function. Maybe adding more detail to the docstring and a typical use-case would help others?

**Comment by mpariente** [comment]
Ok, on it. Thanks for the feedback.

File: asteroid/dsp/overlap_add.py [hunk 1]
```diff
@@ -5,15 +5,24 @@
...
-        reorder_chunks (bool): whether to reorder each consecutive segment.
+        hop_size (int): Segmentation hop size.
+        window (str): Name of the window (see scipy.signal.get_window) used
+            for the synthesis.
+        reorder_chunks (bool): Whether to reorder each consecutive segment.
+            This might be useful when `nnet` is permutation invariant, as
+            source assignements might change output channel from one segment
+            to the next (in classic speech separation for example).
+            Reordering is performed based on the correlation between
+            the overlapped part of consecutive segment.
```
Comment (faroit) [inline asteroid/dsp/overlap_add.py:25]: reads well!

**Review by faroit — APPROVED** [review]

**Comment by mpariente** [comment]
Still needs to be tested in real scenarios but that should work better now.

**Pull request merged**